examples/webpack.config.js

```diff
@@ -84,6 +84,10 @@ module.exports = async (env, argv) => {
             BASEPATH: JSON.stringify(basePath),
         }),
         new SimplestProgressPlugin(),
+        new Dotenv({ 
+            path: "../.env",
```

Comment:
`path: "../.env", // one env for both storybook and examples`

Reply:
added